README.md

```diff
@@ -6,14 +6,78 @@ Guardian is a Terraform actuation and enforcement tool using GitHub actions.
 
 **This is not an official Google product.**
 
+## Guardian Features
+
+NOTE: While some features can be used independently, unless otherwise noted, all features have
```

Comment:
nit: can you adopt the new styling for [Alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)

Reply:
Done